dolphinscheduler-task-plugin/dolphinscheduler-task-mlflow/src/main/java/org/apache/dolphinscheduler/plugin/task/mlflow/MlflowConstants.java

```diff
@@ -86,9 +88,22 @@ private MlflowConstants() {
 
     public static final String MLFLOW_BUILD_DOCKER = "mlflow models build-docker -m %s -n %s --enable-mlserver";
 
-
     public static final String DOCKER_RREMOVE_CONTAINER = "docker rm -f %s";
 
     public static final String DOCKER_RUN = "docker run --name=%s -p=%s:8080 %s";
 
+    public static final String DOCKER_COMPOSE_RUN = "docker-compose up -d";
+
+    public static final String SET_DOCKER_COMPOSE_ENV = "export DS_TASK_MLFLOW_IMAGE_NAME=%s\n" +
+            "export DS_TASK_MLFLOW_CONTAINER_NAME=%s\n" +
+            "export DS_TASK_MLFLOW_DEPLOY_PORT=%s\n" +
+            "export DS_TASK_MLFLOW_CPU_LIMIT=%s\n" +
+            "export DS_TASK_MLFLOW_MEMORY_LIMIT=%s";
+
+    public static final String DOCKER_HEALTH_CHECK_COMMAND = "for i in $(seq 1 20); " +
```

Comment:
I think it's better to let users define the timeout instead of using hard code since the user's server may not be that fast.

Reply:
I think we should prefer to reduce the number of non-essential parameters to make the task plugin easier to use.

But the opinion above is nice. I think we can extend the timeout time, from 1 min to 5min or something else. If the MLflow model service container can not start healthy in 5 min. It's almost certain that something went wrong.

What do you think? @zhongjiajie @SbloodyS

Reply:
Or we can set the timeout to forever If the parameters of Timeout failure are compatible with this approach.
<img width="501" alt="image" src="https://user-images.githubusercontent.com/31528124/172624990-7c5c6dbc-4ea5-4987-a15d-955c2dabcd65.png">

Reply:
I've changed 1 minute to 5 minuites.